package.json

```diff
@@ -77,13 +78,14 @@
       "bit-docs-html-highlight-line": "^0.2.2",
       "bit-docs-tag-demo": "^0.3.0",
       "bit-docs-html-toc": "^0.6.2",
-      "bit-docs-donejs-theme": "^1.6.0"
+      "bit-docs-donejs-theme": "^1.6.0",
+      "@bitovi/calendar-events": "^0.0.8"
```

Comment:
This should go at the top to be consistent with how `devDependencies` is specified.